doc/ref_model/chapters/chapter03.md

```diff
@@ -379,9 +379,9 @@ The SFC management components together with the control components are responsib
 
 Precise Synchronization require specialized card that can be on server or network device motherboard or be part of NIC or both.
 
-OpenStack and Kubernetes clusters use Network Time Protocol [NTP] (RFC 5905: Network Time Protocol Version 4: Protocol and Algorithms Specification, RFC 5906: Network Time Protocol Version 4: Autokey Specification, RFC 5907: Definitions of Managed Objects for Network Time Protocol Version 4 (NTPv4), RFC 5908: Network Time Protocol (NTP) Server Option for DHCPv6) as the default time synchronization for the cluster. That level of synchronization is not sufficient for many network functions. Just like real-time operating system requirement instead of base OS so is precision timing for clock synchronization. Precision Time Protocol version 2 [PTP] (IEEE 1588-2019) is commonly used for Time-Sensitive Networking. This allow synchronization in microsecond range rather than millisecond range that NTP provides.
+OpenStack and Kubernetes clusters use Network Time Protocol (NTP) ([Protocol and Algorithms Specification](RFC 5905: Network Time Protocol Version 4: Protocol and Algorithms Specification), [Autokey Specification](RFC 5906: Network Time Protocol Version 4: Autokey Specification), [Managed Objects](RFC 5907: Definitions of Managed Objects for Network Time Protocol Version 4 (NTPv4)), and [Server Option for DHCPv6](RFC 5908: Network Time Protocol (NTP) Server Option for DHCPv6)) as the default time synchronization for the cluster. That level of synchronization is not sufficient for many network functions. Just like real-time operating systems instead of base OS, so is precision timing for clock synchronization. Precision Time Protocol version 2 [PTP](IEEE 1588-2019) is commonly used for Time-Sensitive Networking. This allow synchronization in microsecond range rather than millisecond range that NTP provides.
```

Comment:
Option 1: plain text references:
```suggestion
OpenStack and Kubernetes clusters use Network Time Protocol (NTP) (RFC 5905: Network Time Protocol Version 4: Protocol and Algorithms Specification, RFC 5906: Network Time Protocol Version 4: Autokey Specification, RFC 5907: Definitions of Managed Objects for Network Time Protocol Version 4 (NTPv4), and RFC 5908: Network Time Protocol (NTP) Server Option for DHCPv6) as the default time synchronization for the cluster. That level of synchronization is not sufficient for many network functions. Just like real-time operating systems instead of base OS, so is precision timing for clock synchronization. Precision Time Protocol version 2 (PTP) (IEEE 1588-2019) is commonly used for Time-Sensitive Networking. This allow synchronization in microsecond range rather than millisecond range that NTP provides.
```